styleguide/writing-style.md

```diff
@@ -0,0 +1,183 @@
+
+Aim to use active voice in instructional step-by-step content and avoid passive voice.\
+Passive voice is useful when you want to emphasize the object, for example when describing the consequence of an activity in the previous sentence. However, stick to active voice for such sentences unless the object requires emphasis.
+
+> [!TIP]
+> While identifying active/passive voice usage can sometimes be tricky, as a general principle, if you find yourself writing "by _verb/action word_" within a lot of your sentences, try flipping the subject and object parts of these sentences around when re-writing them.
+
+### Gender
+
+Always use ‘they’, never use ‘he’ or ‘she’.
+More info on writing about pronouns and in both the [Google developer documentation style guide](https://developers.google.com/style/pronouns#gender-neutral-pronouns) and [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/grammar/nouns-pronouns#pronouns-and-gender) on pronouns.
+
+## Style and formatting
+
+This section covers the matters that go beyond language and provides guidelines for consistency in writing with a unified look.
+
+### Consistency
+
+Keep your writing consistent with itself and other docs. This means abbreviations, capitalization, hyphens, names of UI elements, etc.
+
+### Headings
+
+Use sentence case in all headings:
+
+_"The quick brown fox jumps over the lazy dog."_
+
+The standard case used in English prose. That is, only the first word is capitalized, except for proper nouns and other words which are generally capitalized by a more specific rule.
+
+Do not create multi-sentence headings, or add full stops/periods or other punctuation at the end of a heading.
+
+Refer to [Headings in the Markdown syntax guide](markdown-syntax-style.md#headings) for details on how to implement headings in Markdown.
+
+### Product Names (and product features)
+
+Only use Title Case (initial capital letters) for the name of the product (product names), and match capital letter usage in product names as an organization would use them in these products - for example:
+
+* 'Docker Compose overview' and not 'Docker Compose Overview' 
+* 'GitHub organization' and not 'GitHub Organization'
+
+> [!NOTE]
+> Avoid using Title Case for product features, that is, when mentioning or describing them in documentation.
+
+### UI elements
+
+UI elements should be formatted in italics. For example:
+
+To get your agent token, navigate to _Agents_, then select _Reveal Agent Token_.
+
+> [!NOTE]
+> Match the capitalization used in the Buildkite interface, even if title (or any other) case has been used for product features.
+
+Refer to [UI elements in the Markdown syntax guide](markdown-syntax-style.md#ui-elements) for details on how to write and present UI elements in the Buildkite docs.
+
+### Lists (bullet lists and numbered steps)
+
+Capitalize the first word; no full stops at the end if it's not a full sentence. If it's a full sentence, give it a full stop.
```

Comment:
Nitpick: Semicolons are a bit of a smell to me that the sentence is more complicated than it needs to be. Consider reworking.

Reply:
OK, maybe I'll use the em dash approach instead?

Reply:
On another note, I'm changing 'full stop' to 'full stop/period'.

In the US, from my personal experience, many people haven't heard of the term 'full stop', until you tell them that it's a 'period'.

On yet another note, and I'm not sure how far we're prepared to go with this one, in US English, if there are quotation marks around a final or final set of words in a sentence, the final punctuation character is written before the final quotation mark and not after it, as is the case with British/Australian English.

For example:

* In US English, this sentence would be written "like this."

* In British/Australian English, this sentence would be written "like this".

This discrepancy results from an historical book on writing style, created around the time of typeset printing presses, when placing a period block at the end of a sentence would damage these printing presses more easily, and instead, placing the period before the final character—that is, the quotation mark—would help preserve them. I think once these presses were automated and had the print in embossed plates, this wasn't such an issue. A newer book on writing style came out after these newer presses were produced and switched the order of the final punctuation letter. However, this later style guide was only produced in Britain and its former colonies.

Reply:
Michael and I discussed - add word '/period'.